cylc/flow/scripts/trigger.py

```diff
@@ -96,7 +104,16 @@ def get_option_parser() -> COP:
         multiworkflow=True,
         argdoc=[FULL_ID_MULTI_ARG_DOC],
     )
+
     add_flow_opts(parser)
+
+    parser.add_option(
+        "--on-resume",
+        help=r"Run triggered tasks once a paused workflow is resumed.",
```

Comment:
```suggestion
        help=(
            "If the workflow is paused, wait until it is resumed before "
            "running the triggered task(s)."
        ),
```

Reply:
```suggestion
        help=(
            "If the workflow is paused, wait until it is resumed before "
            "running the triggered task(s). DEPRECATED - this may be "
            "removed at Cylc 8.5."
        ),
```

Reply:
(can probably replace the "may" with "will")

Reply:
I'm wary of making promises (how many "final" Cylc 7 release are we at?) 😅

Reply:
Understood, but we can amend that in the following release in the, hopefully, unlikely event that we need to delay it.

Reply:
```suggestion
        help=(
            "If the workflow is paused, wait until it is resumed before "
            "running the triggered task(s). DEPRECATED - this will be "
            "removed at Cylc 8.5."
        ),
```

Reply:
OK going with "will".